AV-101448 - Documentation update all datasources (#328)
couchbasecloud/terraform-provider-couchbase-capella: docs/data-sources/audit_log_settings.md

page_title: "couchbase-capella_audit_log_settings Data Source - terraform-provider-couchbase-capella"
subcategory: ""
description: |-
  Data source to retrieve audit log configuration settings for a Capella cluster. These settings control which audit events are logged and which users are excluded from audit logging.
---

# couchbase-capella_audit_log_settings (Data Source)

> Data source to retrieve audit log configuration settings for a Capella cluster. These settings control which audit events are logged and which users are excluded from audit logging.

Reviewer: ```suggestion
The data source to retrieve audit log configuration settings for an operational cluster. These settings control which events are logged and which users are excluded.
```
